tests/package_managers/BUILD

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@bazel_tools//tools/build_rules:test_rules.bzl",
+    "rule_test",
+    "file_test",
+)
+load("//package_managers:download_pkgs.bzl", "download_image_pkgs")
+
+download_image_pkgs(
+    name = "test_download_pkgs",
+    base = "//ubuntu:ubuntu_16_0_4_vanilla",
+    packages = [
+        "curl",
+        "netbase",
+    ],
+)
+
+rule_test(
```

Comment:
Nice! I hadn't really seen rule_test and file_test before.

Reply:
done